newIDE/app/src/UI/Menu/MaterialUIMenuImplementation.js

```diff
@@ -91,8 +160,14 @@ export default class MaterialUIMenuImplementation {
             <MenuItem
               dense
               key={'checkbox' + item.label}
-              checked={item.checked}
-              disabled={item.enabled === false}
+              checked={
+                // $FlowFixMe - existence should be inferred by Flow.
```

Comment:
why do you need this all of a sudden?

Reply:
(because it's internal stuff and it works, so not a risk, the next person can improve these if they need)